### PR TITLE
backup: add experimental description for sql br

### PR DESCRIPTION
**Pull request by yusra1007**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

translate doc/cn to en #15708
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15708
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -189,6 +180,18 @@ First, you need to download the binary file of the BR tool. See [download link](
...
 For how to use the command-line tool to perform backup and restore operations, see [Use the BR command-line tool](/br/use-br-command-line-tool.md).
 
+#### Use SQL statements
+
+**Warning:**
+
+The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [issue feedback] on GitHub (/support.md).
+
+In TiDB v4.0.2 and later versions, you can run the BR tool using SQL statements.
+For detailed operations, see the following documents:
```
Comment (dveeden) [inline br/backup-and-restore-tool.md:190]:
```suggestion

For detailed operations, see the following documents:
```

Or merge with like 189.

```
[2024-01-10T09:37:20.189Z] br/backup-and-restore-tool.md: this file has manual line breaks in the following lines:
[2024-01-10T09:37:20.189Z] 
[2024-01-10T09:37:20.189Z] MANUAL LINE BREAKS: L190
```

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -166,29 +166,33 @@ The following are some recommended operations for using BR for backup and restor
 
 ### How to use BR
 
-Currently, the following methods are supported to run the BR tool:
+Currently, the following methods are supported to use BR:
 
-- Use SQL statements
 - Use the command-line tool
+- Use SQL statements
 - Use BR In the Kubernetes environment
```
Comment (ran-huang) [inline br/backup-and-restore-tool.md:173]:
```suggestion
- Use BR in the Kubernetes environment
```

File: br/backup-and-restore-tool.md [hunk 3]
```diff
@@ -166,29 +166,33 @@ The following are some recommended operations for using BR for backup and restor
...
+
+First, you need to download the binary file of the BR tool. See [download link](/download-ecosystem-tools.md#br-backup-and-restore).
+
+For how to use the command-line tool to perform backup and restore operations, see [Use the BR command-line tool](/br/use-br-command-line-tool.md).
+
 #### Use SQL statements
 
+**Warning:**
+
+The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [issue feedback] on GitHub (/support.md).
```
Comment (ran-huang) [inline br/backup-and-restore-tool.md:187]:
```suggestion
> **Warning:**
>
> The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [report it on GitHub](/support.md).
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

**Comment by dveeden** [comment]
This is set to merge into `pingcap:release-4.0`. Is that correct?

**Comment by ran-huang** [comment]
> This is set to merge into `pingcap:release-4.0`. Is that correct?

@dveeden Yes. The Chinese PR was based only on release-4.0

**Review by ran-huang — APPROVED** [review]

**Review by dveeden — APPROVED** [review]

**Pull request merged**